src/table/components/TableWrapper.jsx

```diff
@@ -95,6 +96,9 @@ export default function TableWrapper(props) {
   useDidUpdateEffect(() => {
     if (!keyboard.enabled) return;
 
+    // When Nebula handles keyboard navigation,
+    // a user tabs to the chart and presses Enter or Space,
+    // the focus should be set to the previous focused cell or the first cell in the head
```

Comment:
Not really, it could be set to false as well (keyboard.blur). Something like this is accurete but not sure if it makes it easier to understand
```suggestion
    // When Nebula handles keyboard navigation,
    // and the the table is focused/blurred, the last focused cell should be focused/blurred
```

Reply:
they both blur the table, the argument determines if it should focus the parent (in the client that is the grid cell), at least the way we defined it in client and nebula. `blurCallback` gets it as the `resetFocus` argument in the client. https://github.com/qlik-trial/sense-client/blob/master/web/assets/objects/extension/supernova-wrapper/sn-ext.js#L362